src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/questionWithResponses.tag

```diff
@@ -54,6 +56,20 @@
         </ul>
       </c:if>
 
+      <c:if test="${not isRecipientNameHidden}">
+        <div class="col-sm-12 form-inline mobile-align-left">
+          <label for="input"
+              style="text-indent: 24px">
```

Comment:
This looks short enough to be put in the previous line.

Reply:
Okay